src/muse.c

```diff
@@ -2055,6 +2059,13 @@ mloot_container(
                 break; /* out of takeout_count loop */
         } /* can_carry */
     } /* takeout_count */
+    /* Monster rummaged through a bag but found nothing of interest,possibly
+       because some items vanished. */
+    if (!res && vismon) {
+        pline("%s %s inside %s.", Monnam(mon), 
+                                  mon->mcansee ? "looks" : "feels about",
+                                  contnr_nam);
```

Comment:
The broken up parts should be aligned with the opening paren of pline, like
```c
        pline("%s %s inside %s.", Monnam(mon), 
              mon->mcansee ? "looks" : "feels about", contnr_nam);
```
or
```c
        pline("%s %s inside %s.", Monnam(mon), 
              mon->mcansee ? "looks" : "feels about",
              contnr_nam);
```
etc